docs/main/troubleshooting/troubleshooting-local-storage.md

```diff
@@ -255,5 +255,5 @@ kubectl logs -n openebs -l openebs.io/component-name=openebs-localpv-provisioner
 - [FAQs](../faqs/faqs.md)
 - [Latest Release Notes](../releases.md)
 - [OpenEBS Community](../community.md)
```

Comment:
If community.md is under main folder, then you need to up twice. Please check this link again.

Reply:
These links are already working properly Vishnu. If I up this twice it will have the path like this:

- [OpenEBS Community](../../main/community.md)

which includes "main".